tests/e2e/vLLM/test_vllm.py

```diff
@@ -20,12 +21,19 @@
     logger.warning("vllm is not installed. This test will be skipped")
 
 HF_MODEL_HUB_NAME = "nm-testing"
-TEST_DATA_FILE = os.environ.get("TEST_DATA_FILE", None)
+TEST_DATA_FILE = os.environ.get("TEST_DATA_FILE", "")
+
+
+@pytest.fixture
+def record_config_file(record_testsuite_property: Callable[[str, object], None]):
+    test_data_file_name = TEST_DATA_FILE.split("configs/")[-1]
+    record_testsuite_property("TEST_DATA_FILE_NAME", test_data_file_name)
 
 
 # Will run each test case in its own process through run_tests.sh
 # emulating vLLM CI testing
 @requires_gpu_count(1)
+@pytest.mark.skipif(not TEST_DATA_FILE, reason="must set TEST_DATA_FILE env var")
```

Comment:
I don't think this should be a reason we skip a test case. we should probably just throw a warning or error out

Reply:
Generally speaking, it’s the same general idea as the vllm-based skip – both check if an environmental factor that is required for the test is present.

That said, I’ve removed the new skip entirely.